docs/nodes/docker-images.mdx

```diff
@@ -36,13 +39,13 @@ import TabItem from '@theme/TabItem';
 <Tabs groupId="network">
 <TabItem value="mocha" label="Mocha">
 
-## Mocha Testnet Setup
+### Mocha Testnet Setup
 
 Run the image from the command line:
 
-```bash 
-docker run -e NODE_TYPE=light -e P2P_NETWORK=mocha ghcr.io/celestiaorg/celestia-node:sha-747c9e5 celestia light start --core.ip https://rpc-mocha.pops.one --gateway --gateway.addr 127.0.0.1 --gateway.port 26659 --p2p.network mocha
-```
+<pre><code>
+docker run -e NODE_TYPE=light -e P2P_NETWORK=mocha ghcr.io/celestiaorg/celestia-node:{mochaVersions['node-latest-tag'].slice(1)} celestia light start --core.ip https://rpc-mocha.pops.one --gateway --gateway.addr 127.0.0.1 --gateway.port 26659 --p2p.network mocha
```

Comment:
Did you mean to use the tag here instead of the sha?

Reply:
it now matches the arabica command but just calling out the change

Reply:
yeah, i did. on v0.7.0-rc1, it was correctly tagged again. i will keep an eye on this as the integration is worked in to docs and new versions are released